1.9/api/index.md

```diff
@@ -0,0 +1,38 @@
+---
+post_title: DC/OS API Reference
+nav_title: API Reference
+menu_order: 4
+---
+
+The DC/OS API is a collection of routes backed by [DC/OS Components](/docs/1.9/overview/architecture/components/) made available through [Admin Router](/docs/1.9/overview/architecture/components/#admin-router), an API gateway.
+
+To determine the address of your cluster, see [Cluster Access](/docs/1.9/api/access/).
```

Comment:
Well it's a thing that and API user would need to do to use the API... how should I say that? Should it be at the top of a Routes section?